### DISCUSSION TITLE
Integration HmIP-BDT    (Unterputzdimmer) - wird in HA nicht erkannt

### DISCUSSION THREAD

**Discussion by sfiala03** [SukramJ/aiohomematic#1421] [Ideas]
Ich habe einen HmIP-BDT neu verbaut, aber er wird in der HA-Homematic-integration nicht erkannt. Alle anderen Homematicgeräte sind über HA ansprechbar, aber der Dimmer erscheint überhaupt nicht. 
Es wäre schön, wenn dieses Gerät auch in die Liste der Geräte aufgenommen werden könnte.
Danke für die bisher von euch geleistete Arbeit. Ich bin leider nur "User" und kann evtl. nichts dazubeitragen, dass neue Geräte aufgenommen werden. Oder gibt es eine Anleitung wie man dabei vorzugehen hat? Ich hab dazu nichts gefunden.
Stephan

**Comment by SukramJ** [comment]
ggf. hilft der Aufruf des Service  [clear_cache](https://github.com/danielperna84/custom_homematic#homematicip_localclear_cache) 
gefolgt von einem Neustart.

**Reply by sfiala03** [reply]
Danke, hab ich gleich probiert, hat aber nichts gebracht.
In https://github.com/danielperna84/pyhomematic/blob/master/pyhomematic/devicetypes/actors.py
ist der HmIp-BDT aufgeführt. "HMIP-BDT": IPKeyDimmer, "HmIP-BDT": IPKeyDimmer,
müsste also erkannt werden!?
Gibts noch eine Idee warum nicht?
Danke und Grüße
Stephan

**Reply by danielperna84** [reply]
Das was du da postest ist von der alten Integration. Hier ist das Repository von der neuen Integration. Welche nutzt du? Die mit der configuration.yaml, oder tatsächlich die neue?

**Reply by sfiala03** [reply]
Eigentlich die neue!? Würde ich sagen -  wenn auf die Integration gehe erhalte 
ccu3
Homematic(IP) Local 

und bei Dokumentation werde ich zu dem danielperna84/custom_homematic repo geleitet. Passt also würde ich sagen.

**Reply by SukramJ** [reply]
Der HmIP-BDT wird von Anfang an unterstützt, und mein eigener BDT funktioniert problemlos.
Das Problem ist also auf deinem System.

Aus Erfahrung der letzten Jahre kann ich empfehlen auch mal nur nach dem Gerätetyp (HmIP-BDT) in der Geräteübersicht zu suchen. Es soll schon vorgekommen sein, daß der erwartete Name vom echten Namen abweicht.

Ansonsten wäre es am Einfachsten, wenn Du hier ein Issue aufmachst, und die angefragten Informationen möglichst vollständig ausfüllst, oder die im Issue angefragten Informationen hier möglichst vollständig ergänzt.

**Reply by sfiala03** [reply]
Danke für eure Infos und Bemühungen. Gut zu hören, dass der BDT bei dir funktioniert. Dann muss der Fehler bei mir liegen.
Nach dem Gerätetyp hab ich schon x-mal gesucht und überprüft ob nicht doch aufgetaucht ist, ist er aber nicht.
Ich werde ihn erstmal in der Raspimatic abmelden und dann nochmal als neues Gerät hinzufügen. Vielleicht klappts ja dann in der HA integration auch.. Ich werde berichten und dann falls notwendig deinen Vorschlag mit der Issue befolgen.
Danke und Grüße
Stephan
